plugin/code_actions.py

```diff
@@ -423,10 +414,9 @@ def _is_cache_valid(self, event: dict | None) -> bool:
         view = self.view
         if not view:
             return False
-        region = self._get_region(event)
-        if region is None:
-            return False
-        return actions_manager.menu_actions_cache_key == f"{view.buffer_id()}#{view.change_count()}:{region}"
+        if region := self._get_region(event):
+            return actions_manager.menu_actions_cache_key == f"{view.buffer_id()}#{view.change_count()}:{region}"
+        return False
```

Comment:
I have not checked whether this would affect this particular case but we theoretically we should be checking region against `None` since empty region also evaluates to `False`.

Same below.